src/components/WebcamCapture/ViewFinder.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ViewFinder = props => {
+  const sw = 72; // Stroke width (grey border)
+  const w = 768; // Width (of display)
+  const vfw = 8; // View finder width (white bits)
+  const vfl = Math.min(64 + props.animation * (w / 2 - sw - 64), w / 2 - sw); // View finder length (white bits)
```

Comment:
Is it possible to grab these values programmatically? - e.g. get display width then calculate the rest as percentages

Reply:
The only one that is dynamic is the `w` variable and it would be a lot of hassle to grab it. considering it's only going to run on the iPad I don't think it's worth the hassle.